package.json

```diff
@@ -11,7 +11,7 @@
     "lint:sass": "NODE_ENV=production sass-lint -v --max-warnings 0",
     "postinstall": "npm prune && npm run build",
     "starterapp": "webpack-dev-server --config webpack/starterApp.js --devtool eval --progress --colors --hot --history-api-fallback",
-    "test": "NODE_ENV=test karma start",
+    "test": "karma start",
```

Comment:
To answer my own question: because Karma runs the tests in the browser, there is no such thing as `NODE_ENV` there, so this wasn't necessary.